<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMAddPerson</class>
 <widget class="QWidget" name="IMAddPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMAddPerson</string>
  </property>
  <widget class="QLabel" name="mLabelBK">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>700</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mPButtonFindPerson">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>找人</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mPButtonFindGroup">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>30</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>找群</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mPButtonFindAppNo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>应用号服务号</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>65</y>
     <width>700</width>
     <height>385</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
